### PR TITLE
Add Arabic Date

### PR DESCRIPTION
**Pull request by F-xobj**

### CODE REVIEW

**Comment by Marak** [comment]
@F-xobj -

Thank you for your contribution.

I don't really speak Arabic. Could we get another person who speaks the language to confirm these commits?

I checked the Git history and think @thaerlabs or @anas-qa can assist.

**Comment by F-xobj** [comment]
> @F-xobj -
> 
> Thank you for your contribution.
> 
> I don't really speak Arabic. Could we get another person who speaks the language to confirm these commits?
> 
> I checked the Git history and think @thaerlabs or @anas-qa can assist.

Sure . will be happy if they see it .  @thaerlabs ,@anas-qa  can you please confirm these changes ?
if all is good , i will add more commits . 
Thanks all .

File: lib/locales/ar/address/state.js [hunk 1]
```diff
@@ -48,5 +70,44 @@ module["exports"] = [
...
+  "الزرقاء",
+  "إربد",
+  "العقبة",
+  "السلط",
+  "المفرق",
+  "جرش",
+  "معان",
+  "عجلون",
+  "حلب",
+  "مشق",
```
Comment (naderalfakesh) [inline lib/locales/ar/address/state.js:84]: Please change "مشق", into "دمشق",

File: lib/locales/ar/address/state.js [hunk 2]
```diff
@@ -48,5 +70,44 @@ module["exports"] = [
...
+  "حمص",
+  "اللاذقية",
+  "حماة",
+  "طرطوس",
+  "ير الزور",
```
Comment (naderalfakesh) [inline lib/locales/ar/address/state.js:89]: Please change   "ير الزور", into 
  "دير الزور",

File: lib/locales/ar/address/state.js [hunk 3]
```diff
@@ -48,5 +70,44 @@ module["exports"] = [
...
+  "السويداء",
+  "لحسكة",
```
Comment (naderalfakesh) [inline lib/locales/ar/address/state.js:91]: "لحسكة",
ino 
"الحسكة",

**Review by naderalfakesh — CHANGES_REQUESTED** [review]
Hi 
Just fix these three typos and I can give approve feedback. 
I am a native speaker. 
Best regards...

**Comment by F-xobj** [comment]
@naderalfakesh can you re-review again, thanks!

**Comment by fityanos** [comment]
@F-xobj LGTM 🎯

**Comment by Marak** [comment]
Thank you for your contributions @F-xobj and @fityanos and @naderalfakesh 

Merged.

**Pull request merged**